generator/src/components/FormSelectGroup/index.svelte

```diff
@@ -0,0 +1,31 @@
+<script context="module" lang="ts">
+  export type Option = {
+    id: string
+    label: string
+  }
+</script>
+
+<script lang="ts">
+  import Svelecte from "svelecte"
+
+  export let name: string
+  export let label: string
+  export let placeholder: string = ""
+  export let options: Option[] = []
+</script>
+
+<div class="form-group">
+  <label for={name}>{label}</label>
+  <Svelecte {options} {placeholder} />
```

Comment:
If you don't add `{name}` how does the `label` work?

Reply:
label element requires name field

Reply:
I'm asking, how does clicking on `<label>` work to select the correct element, which is what `<label>` is used for, when the `Svelecte` does not have `{name}` argument?
![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/4a2ab6b2-c07e-4ef2-b808-19902e5f8fb3)